ansible/playbooks/aix.yml

```diff
@@ -717,6 +717,30 @@
             # Environment variables don't work as part of command
             - skip_ansible_lint
 
+      #########################
+      # Install Asian Locales #
+      #########################
+        - name: Transfer locales installp to remote host
+          copy:
+            src: /Vendor_Files/aix/asian.tar.gz
```

Comment:
Can you document where/how to obtain these files for future reference when updating is required.

Reply:
Thank you for your review. Yes, I added fileset list which should contain the tar file.

Reply:
I expect these filesets are picked from AIX base OS media. I can create the file if the target AIX's version (TL/SP) is identified.
I appreciate any advice.

Reply:
For what version is this file generated?

Reply:
Thank you for the comment. 
My test environment is AIX7.1TL5.

I think it's better the target filesets are stored in /Vendor_Files/aix without packing. I updated this pull request.

I don't know how I can request to store files in /Vendor_Files. Could you kindly advice that?

I don't think the AIX filesets can be downloaded (We can download update images though). I believe they should be copied from AIX base OS media.

Reply:
@toshiona We'd need to be 100% certain we're not violating any license agreements if we do that. I'm nervous about copying stuff from the AIX install media across to our servers (this redistributing them)
Can you separate the AIX bits into a separate PR so we can get the Linux stuff merged ASAP.